Vagrantfile

```diff
@@ -1,11 +1,17 @@
+$vpupp = <<-'SCRIPT'
```

Comment:
It's great to see you learning new techniques like this, but can we keep it inline instead of using a heredoc?

Here is an example: https://github.com/oxr463/pentesting-vm/blob/master/Vagrantfile#L14